cross-media-measurement/src/main/cc/wfa/measurement/common/crypto/liquid_legions_v2_encryption_utility.cc

```diff
@@ -802,17 +798,30 @@ CompleteExecutionPhaseTwoAtAggregator(
 
   // Estimates reach.
   int64_t active_register_count = tuple_counts - blinded_histogram_noise_count;
```

Comment:
Isn't this the number of _non-empty_ registers?
(IIUC, in the write-up, we use _active_ to mean non-empty and non-destroyed)

Reply:
I was using active to refer to all registers that has value in the sketch.

Renamed to non-empty-register here.